secantTests/Util/RecoveryPhraseStorageTests.swift

```diff
@@ -0,0 +1,227 @@
+//
+//  RecoveryPhraseStorageTests.swift
+//  secantTests
+//
+//  Created by Lukáš Korba on 10.03.2022.
+//
+
+import XCTest
+@testable import secant_testnet
+
+extension KeyStoringError {
+    var debugValue: String {
+        switch self {
+        case .alreadyImported: return "alreadyImported"
+        case .uninitializedWallet: return "uninitializedWallet"
+        case .storageError: return "storageError"
+        }
+    }
+}
+
+class RecoveryPhraseStorageTests: XCTestCase {
+    let birthdaySet = BlockHeight(12345678)
+    let recoveryPhraseSet = "one two three"
+    var storage: RecoveryPhraseStorage?
+
+    override func setUp() {
+        super.setUp()
+        storage = RecoveryPhraseStorage()
+        deleteData(forKey: "zcashRecoveryPhrase")
+        deleteData(forKey: "zcashBirthday")
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        storage = nil
+    }
+    
+    func testRecoveryPhraseStoredSucessfuly() throws {
+        do {
+            try storage?.importPhrase(bip39: recoveryPhraseSet)
+            guard let data = data(forKey: "zcashRecoveryPhrase") else {
```

Comment:
Can this test not access `Constants.zcashRecoveryPhrase` so that we don't have to use a string here?

Reply:
done